react/features/base/tracks/loadEffects.native.js

```diff
@@ -7,5 +7,5 @@
  * @returns {Promsie} - A Promise which resolves when all effects are created.
  */
 export default function loadEffects(store: Object): Promise<any> { // eslint-disable-line no-unused-vars
-    return Promise.resolve();
+    return Promise.resolve([]);
```

Comment:
maybe worth updating JSDoc for the return type?

Reply:
would flow help here?

Reply:
good point, i'll fix it in a jiffy